index.html

```diff
@@ -40,6 +46,26 @@
             openmct.install(openmct.plugins.Espresso());
             openmct.install(openmct.plugins.Generator());
             openmct.install(openmct.plugins.UTCTimeSystem());
+
+            /*
+                Will be installed by default...somehow
+             */
+            //openmct.install(ConductorService());
+
+            var ONE_MINUTE = 60 * 1000;
+            var ONE_YEAR = 365 * 24 * 60 * 60 * 1000;
+
+            // Will also provide a default configuration based on enabled time
+            // systems and tick sources.
+            openmct.install(openmct.plugins.Conductor({
```

Comment:
This is good in that it becomes really clear what the Conductor plugin is, and that wasn't very clear before.  Now, the conductor plugin can be summed up in a screenshot:

![screen shot 2017-04-05 at 11 22 48 am](https://cloud.githubusercontent.com/assets/226503/24720555/4c448f9a-19f2-11e7-9b67-d36b243d8fc1.png)
(sorry for outdated screenshot).

In that context, the configuration makes sense-- you are configuring that view and what the options will be in that view.  

On the other hand, the setting of defaults doesn't make sense within that view-- I'd expect that to be part of the Time API, as if the Conductor plugin isn't installed, we still need a way to set defaults for the TimeAPI.